lib/ReactTags.js

```diff
@@ -60,7 +60,14 @@ class ReactTags extends React.Component {
     const { query, selectedIndex } = this.state
 
     // when one of the terminating keys is pressed, add current query to the tags.
-    if (e.keyCode === KEYS.ENTER || e.keyCode === KEYS.TAB) {
+    let shouldAdd = e.keyCode === KEYS.ENTER || e.keyCode === KEYS.TAB
+
+    // if delimiters were explicitly provided, reasses based on delimiters.
+    if (this.props.delimiters) {
```

Comment:
Personally I'd set `ReactTags.defaultProps.delimiters = [KEYS.ENTER, KEYS.TAB]` instead, then you're only checking one place. Probably worthwhile to verify that at least one item is present as well.